src/sonic-config-engine/tests/test_cfggen_from_yang.py

```diff
@@ -194,7 +192,8 @@ def test_acl_rule(self):
             "EVERFLOW|Rule2": {
                 "DST_IP": "192.169.10.1/32",
                 "SRC_IP": "10.10.1.1/16",
-                "IP_TYPE": "IPV4"
+                "IP_TYPE": "IPV4",
+                "PRIORITY": "101"
```

Comment:
hi @nazariig could you clarify why add PRIORITY for everflow in this PR?

Reply:
@StormLiangMS this is hotfix - was added to prevent test failures. Aligned with master